front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -1,188 +1,194 @@
-import { OsrdConfState, PointOnMap } from 'applications/operationalStudies/consts';
-import { Allowance } from 'common/api/osrdEditoastApi';
-import { omit } from 'lodash';
 import { createStoreWithoutMiddleware } from 'Store';
-import { defaultCommonConf } from '..';
-import { simulationConfSliceType } from '../../simulationConf';
-import { stdcmConfSliceType } from '../../stdcmConf';
-import commonConfBuilder from './commonConfBuilder';
+
+import type { OsrdConfState, PointOnMap } from 'applications/operationalStudies/consts';
+
+import type { Allowance } from 'common/api/osrdEditoastApi';
+
+import { defaultCommonConf } from 'reducers/osrdconf/osrdConfCommon';
+import type { stdcmConfSliceType } from 'reducers/osrdconf/stdcmConf';
+import infraTestBuilder from 'reducers/infra/__tests__/infraTestBuilder';
+import type { operationalStudiesConfSliceType } from 'reducers/osrdconf/operationalStudiesConf';
+import commonConfBuilder from 'reducers/osrdconf/osrdConfCommon/__tests__/commonConfBuilder';
 
 function createStore(
-  slice: simulationConfSliceType | stdcmConfSliceType,
+  slice: operationalStudiesConfSliceType | stdcmConfSliceType,
   initialStateExtra: Partial<OsrdConfState> = {}
 ) {
   return createStoreWithoutMiddleware({
     [slice.name]: { ...defaultCommonConf, ...initialStateExtra },
   });
 }
 
-const testCommonConfReducers = (slice: simulationConfSliceType | stdcmConfSliceType) => {
+const testSpeedLimitByTag = (
+  slice: operationalStudiesConfSliceType | stdcmConfSliceType,
+  newTag: string | null
+) => {
+  const store = createStore(slice);
+  store.dispatch(slice.actions.updateSpeedLimitByTag(newTag));
+  return store.getState()[slice.name];
+};
+
+const testCommonConfReducers = (slice: operationalStudiesConfSliceType | stdcmConfSliceType) => {
   const testDataBuilder = commonConfBuilder();
+  const infraTestDataBuilder = infraTestBuilder();
+  let defaultStore: ReturnType<typeof createStore>;
+
+  beforeEach(() => {
+    defaultStore = createStore(slice);
+  });
```

Comment:
Very nice to have factorized the creation of the store here. But why did you rename store by defaultStore ? I think store was a good name (simple remark)

Reply:
We need to create a custom store in the test ('should handle updateOriginTime').
So we give defaultStore here.